### ISSUE TITLE
During checkout: missing successful global message when user selects different Payment Method

### ISSUE THREAD

**Issue by developpeurweb** [SAP/spartacus#13729]
**Describe the bug**
- Screen reader users wouldn't be able to tell if they have switched/selected an additional Payment Method during checkout.

**Tell us the version of Spartacus**
- Library version: 4.0.0

**To Reproduce**

1. Log in as a registered user
2. Add items to the cart and start the checkout process.
3. Open any Screen Reader (VO, NVDA, JAWS).
4. Switch Payment Method from default to an additional one.
5. Listen to Screen Reader feedback
6. Hear the absence of confirmation a change has happened.

**Expected behavior**
- Screen reader users will hear a successful confirmation message saying they have changed/selected a different Payment Method. E.g _"Payment Method selection successfully changed"._
- For `MSG_TYPE_CONFIRMATION` use `aria-live="polite" aria-atomic="true"`

**Screenshots**

Message suggestion in the screenshot below, please validate phrasing with @Manurod2018 or @vekterdude

![pm-change-2021-09-15_10-22-53](https://user-images.githubusercontent.com/15113219/133455057-a1a8c463-b3bb-456c-a480-62eab0fd8d8a.jpg)

**Comment by BenjaminEmiliani** [comment]
*NOTE* Any Global Message emitted is now read by Screen Readers (recent change), the only change required to solve this ticket is to trigger a Global Message upon switching/selecting an additional Payment Method during checkout.

**Comment by BenjaminEmiliani** [comment]
Hold, need to confirm with Blamed if these changes would still apply after code splitting checkout.

**Comment by KateChuen** [comment]
Tested on develop branch of spartacus using installation script
More extensive testing (on other screenreaders and devices) will be done with RC.
![switchPayment1](https://user-images.githubusercontent.com/45409515/145105387-9f6929b3-c13a-453f-9834-0d018a6a8ece.png)
![switchPayment](https://user-images.githubusercontent.com/45409515/145105388-180bd957-c981-4a6b-ad44-75ce1c4ecdf2.png)
